paddle/fluid/operators/CMakeLists.txt

```diff
@@ -36,15 +36,15 @@ endif()
 
 register_operators(EXCLUDES warpctc_op conv_fusion_op)
 
-# warpctc_cudnn need cudnn 7 above
+# ops need cudnn 7 above
 if (WITH_GPU)
-    if (${CUDNN_MAJOR_VERSION} VERSION_LESS 7)
+    if (NOT ${CUDNN_MAJOR_VERSION} VERSION_LESS 7)
```

Comment:
actually, this means, if cudnn ver > 7, use sources `warpctc_cudnn_op.cc` else use other sources.